flair/models/sequence_tagger_model.py

```diff
@@ -514,26 +515,28 @@ def _calculate_loss_old(self, features, lengths, tags) -> float:
             score /= len(features)
             return score
 
-    def _obtain_labels(self, feature, sentences) -> List[List[Label]]:
+    def _obtain_labels(self, feature, sentences) -> (List[List[Label]], List[List[List[Label]]]):
```

Comment:
I think it's not very transparent what the return values correspond to. Maybe we can add some explanation along the lines of:

```py
"""
Returns a tuple of two lists: 
    - The first list corresponds to the most likely `Label` per token in each sentence.
    - The second list contains a probability distribution over all `Labels` for each token in a sentence for all sentences.
"""
```py